runtime/README.md

```diff
@@ -1,7 +1,22 @@
-# Caliptra Runtime Firmware v1.1
+# Caliptra Runtime Firmware v2.0
```

Comment:
We should talk more about code size & ICCM size. I think we may have a hard time fitting all of this in 128KiB of SRAM.

Reply:
Agreed.